rank/data/tests/images.json

```diff
@@ -0,0 +1,142 @@
+                "ratings": [
+                    "gzv2hhgy"
+                ],
+                "description": "image id"
+            },
+            {
+                "searchTerms": "kmebmktz",
+                "ratings": [
+                    "gzv2hhgy"
+                ],
+                "description": "search for work ID and get associated images"
+            },
+            {
+                "searchTerms": "L0033046",
+                "ratings": [
+                    "gzv2hhgy"
+                ],
+                "description": "miro ID"
+            }
+        ],
+        "metric": {
+            "precision": {
+                "relevant_rating_threshold": 3,
+                "k": 1
+            }
+        }
+    },
+    {
+        "id": "recall",
+        "description": "TBD",
+        "eval": "equalTo1",
+        "cases": [
+            {
+                "searchTerms": "horse battle",
+                "ratings": [
+                    "ud35y7c8"
+                ]
+            },
+            {
+                "searchTerms": "everest chest",
+                "ratings": [
+                    "bt9yvss2",
+                    "erth8sur",
+                    "fddgu7pe",
+                    "qbvq42t6",
+                    "u6ejpuxu",
+                    "xskq2fsc",
+                    "prrq5ajp",
+                    "zw53jx3j"
+                ]
+            },
+            {
+                "searchTerms": "Frederic Cayley Robinson",
+                "ratings": [
+                    "avvynvp3",
+                    "b286u5hw",
+                    "dey48vd8",
+                    "g6n5e53n",
+                    "gcr92r4d",
+                    "gh3y9p3y",
+                    "vmm6hvuk",
+                    "z894cnj8",
+                    "cfgh5xqh",
+                    "fgszwax3",
+                    "hc4jc2ax",
+                    "hfyyg6y4",
+                    "jz4bkatc",
+                    "khw4yqzx",
+                    "npgefkju",
+                    "q3sw6v4p",
+                    "z7huxjwf",
+                    "dkj7jswg",
+                    "gve6469u",
+                    "kyw8ufwn",
+                    "r49f89rq",
+                    "sptagjhw",
+                    "t6jb62an",
+                    "tq4qjedt",
+                    "xkt8av46",
+                    "yh6evjnu",
+                    "dvg8e7h5",
+                    "knc95egk",
+                    "th8c2wan"
+                ]
+            }
+        ],
+        "metric": {
+            "recall": {
+                "relevant_rating_threshold": 3,
+                "k": 30
+            }
+        }
+    },
+    {
+        "id": "alternative-spellings",
+        "description": "TBD",
+        "eval": "equalTo1",
+        "cases": [
+            {
+                "searchTerms": "arbeiten",
+                "ratings": [
+                    "sr4kxmk3",
+                    "utbtee43"
+                ]
```

Comment:
Similar to above, and esp because metadata changes – it'd be good to know what you thought the similar text was in the image.

It's not clear to me what it is in either case – `sr4kxmk3` literally contains "Arbeiten" in the title which is just cased differently, and I have no idea what's being matched on `utbtee43`.